### DISCUSSION TITLE
the problem with fonts

### DISCUSSION THREAD

**Discussion by Artyom-Murzinov** [mozman/ezdxf#1197] [Issues]
I specify the font directory, but it either does not pass through this directory or does not read fonts, all fonts are ttf, on Mac os only those that were operating systemically, and the same trouble with debian!
![Снимок экрана 2024-11-11 в 20 32 14](https://github.com/user-attachments/assets/c9f44243-acae-4bd5-9aa3-d2556af83024)

**Comment by mozman** [comment]
You have to create a config file as shown in the documentation. Changing the configuration on the fly is not possible.

**Reply by Artyom-Murzinov** [reply]
Something still doesn't work out.
![Снимок экрана 2024-11-12 в 18 56 44](https://github.com/user-attachments/assets/f9876592-7c84-4b03-98d0-ad68c6631026)
